shoppingfluxexport.php

```diff
@@ -1126,6 +1126,10 @@ private function _getCombinaisons($product, $configuration, $link, $carrier, $fi
             $ret .= '<'.$this->_translateField('mpn').'><![CDATA['.$combination['reference'].']]></'.$this->_translateField('mpn').'>';
             $ret .= '<'.$this->_translateField('combination_link').'><![CDATA['.$link->getProductLink($product).$product->getAnchor($id, true).']]></'.$this->_translateField('combination_link').'>';
 
+            if (method_exists('Product', 'combinationFeaturesAsXml')) {
```

Comment:
The incoming 4.2 allow to add fields override in xml feed.

Its called here: https://github.com/shoppingflux/shoppingfluxexport/pull/48/files#diff-f43af2910e0618bcfcdc0ac20edddfecR1099

And the method: https://github.com/shoppingflux/shoppingfluxexport/pull/48/files#diff-f43af2910e0618bcfcdc0ac20edddfecR2096

Fields can be configured in module admin.

This can solve your problem ?